docker/.env.example

```
@@ -0,0 +1,139 @@
+POSTGRES_PORT=5432
+POSTGRES_USER=postgres
+# default user is postgres
+
+
+############
+# Supavisor -- Database pooler
+############
+# Port Supavisor listens on for transaction pooling connections
+POOLER_PROXY_PORT_TRANSACTION=6543
+# Maximum number of PostgreSQL connections Supavisor opens per pool
+POOLER_DEFAULT_POOL_SIZE=20
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The POOLER_DEFAULT_POOL_SIZE key should go before the POOLER_PROXY_PORT_TRANSACTION key

(UnorderedKey)